packages/checkbox/src/index.mts

```diff
@@ -52,7 +53,7 @@ function check(checked: boolean) {
   };
 }
 
-function renderItem<Value>({ item, isActive }: { item: Item<Value>; isActive: boolean }) {
```

Comment:
If we want to change this to `isActive` it needs to be changed in the `Layout` type. It also needs to be consistent between `select` and `checkbox` - presently `select` uses `active` while `checkbox` uses `isActive` - I guess I missed making this consistent